shared/src/persistence/dynamo/notifications/saveUserConnection.js

```diff
@@ -19,10 +19,11 @@ exports.saveUserConnection = async ({
   const TIME_TO_EXIST = 60 * 60 * 24;
   return await put({
     Item: {
+      connectionId,
       endpoint,
-      gsi1pk: connectionId,
-      pk: `connections-${userId}`,
-      sk: connectionId,
+      gsi1pk: `connection|${connectionId}`,
+      pk: `user|${userId}`,
+      sk: `connection|${connectionId}`,
       ttl: Math.floor(Date.now() / 1000) + TIME_TO_EXIST,
```

Comment:
Why is the TTL divided by 1000? (and what units of time is the TTL in? - TIME_TO_EXIST is seconds in a day, is TTL also in seconds?)

Reply:
Also, are there any tests that try to connect as a user within the TTL and outside the TTL?

Reply:
TTL is in seconds, since `Date.now()` returns the [current timestamp in milliseconds](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now). I don't _believe_ there are any specific outside/inside TTL unit tests, because the implementation for `put()` is mocked/stubbed out on the surface level of `put` itself (i.e. just its definition).

Reply:
Furthermore, the in/out logic should be on the interactor (or use case) itself (i.e. if the user's connection should _updated_, or in this case, saved/created vs. destroyed). This saves the connection for the current moment in time. 

This is triggered through notifications fired via websocket for user inactivity on the UI.